ROADMAP.md

```diff
@@ -19,16 +40,13 @@ Reference: [Control plane issues](https://github.com/kserve/kserve/issues?q=is%3
 
 ## Objective: "Graduate KServe Python SDK to 1.0“
 
-- Improve KServe Python SDK dependency management with Poetry
```

Comment:
Is it worth mentioning the improvement around the decoupling of the storage-initializer dependencies from KServe? https://github.com/kserve/kserve/issues/3489